katharsis-meta/src/main/java/io/katharsis/meta/MetaModule.java

```diff
@@ -8,7 +8,7 @@
 import io.katharsis.core.internal.utils.PreconditionUtil;
 import io.katharsis.legacy.registry.DefaultResourceInformationBuilderContext;
 import io.katharsis.meta.internal.MetaRelationshipRepository;
-import io.katharsis.meta.internal.MetaResourceRepository;
+import io.katharsis.meta.internal.MetaResourceRepositoryImpl;
```

Comment:
yes, naming is a bit difficult in that area since there is already a MetaRepository and a MetraResourceRepository. I was also looking for a complete other name for the entire project other than "meta", but did not really come up with something (Descriptor, Information, Model would be candidates, but I prefer meta).  To some degree it is also not a DefaultMetaResourceRespository either because it likely/hopefully will remain the only implementation. But I'm ok with either.